course_contents/2_intro_to_python/notes/03. Comprehensions.md

```diff
@@ -0,0 +1,99 @@
+### The list comprehension
+- the list comprehension allows us to create a list without manually appending objects or running loops
```

Comment:
I would reword this to:
>We can use a list comprehension to create a list from another iterable

Reply:
Yup, that sounds better and it's shorter